### PR TITLE
Refonte partie Administration + Complétion Controller API UniFolio

### PR DESCRIPTION
**Pull request by CyndelHerolt**

### CODE REVIEW

**Review by Dannebicque — COMMENTED** [review]
OK

**Review by Dannebicque — COMMENTED** [review]
OK

**Review by Dannebicque — APPROVED** [review]
Accepté

File: templates/administration/block_admin/_etudiant.html.twig [hunk 1]
```diff
@@ -2,54 +2,65 @@
...
+                        <p>Gestion des absences, des notes, des étudiants, des fins de semestres,
+                            des
+                            stages</p>
+                    </div>
+                    <hr>
+                    {% for semestre in user_data.semestres|filter(semestre => semestre.actif == true) %}
+
+                        {% if semestre.annee.optAlternance == false %}
+
+                            <a href="{{ path('administration_etudiant_parcours_semestre_index', {'semestre': semestre.id } )}}">{{ semestre.displayAvecTypeDiplome }}</a>
```
Comment (Dannebicque) [inline templates/administration/block_admin/_etudiant.html.twig:45]: href="{{ path('administration_semestre_index', {semestre:semestre.id}) }}"

File: templates/administration/index.html.twig [hunk 2]
```diff
@@ -31,16 +31,16 @@
...
 
-            {% if user_data.departement.optStage %}
-                {{ include('administration/block_admin/_stage.html.twig') }}
-            {% endif %}
-            {{ include('administration/block_admin/_alternance.html.twig') }}
-            {{ include('administration/block_admin/_projet.html.twig') }}
-            {{ include('administration/block_admin/_qualite.html.twig') }}
-            {{ include('administration/block_admin/_article.html.twig') }}
-            {{ include('administration/block_admin/_document.html.twig') }}
+{#            {% if user_data.departement.optStage %}#}
```
Comment (Dannebicque) [inline templates/administration/index.html.twig:34]: Pourquoi commenté ?

**Comment by CyndelHerolt** [comment]
Simple oubli de ma part je crois, je revérifie de mon coté Envoyé à partir de Courrier pour Windows De : DavidEnvoyé le :Saturday, June 3, 2023 10:04 PMÀ : Dannebicque/intranetV3Cc : CyndelHerolt; AuthorObjet :Re: [Dannebicque/intranetV3] Refonte partie Administration + Complétion Controller API UniFolio (PR ***@***.*** commented on this pull request.In templates/administration/block_admin/_etudiant.html.twig:> -                        {{ 'adm.statistiques.menu'|trans }}</a>++                <div class="col-lg-3 col-10 d-grid d-block mb-2 etudiant_bloc">+                    <div class="admin_links_bloc_header">+                        <i class="fa-solid fa-user-graduate fa-2x"></i>+                        <h4>Formation Initiale</h4>+                        <p>Gestion des absences, des notes, des étudiants, des fins de semestres,+                            des+                            stages</p>+                    </div>+                    <hr>+                    {% for semestre in user_data.semestres|filter(semestre => semestre.actif == true) %}++                        {% if semestre.annee.optAlternance == false %}++                            <a href="{{ path('administration_etudiant_parcours_semestre_index', {'semestre': semestre.id } )}}">{{ semestre.displayAvecTypeDiplome }}</a>                            href="{{ path('administration_semestre_index', {semestre:semestre.id}) }}"In templates/administration/index.html.twig:> @@ -31,16 +31,16 @@             {{ include('administration/block_admin/_ppn.html.twig') }}             {{ include('administration/block_admin/_edt.html.twig') }} -            {% if user_data.departement.optStage %}-                {{ include('administration/block_admin/_stage.html.twig') }}-            {% endif %}-            {{ include('administration/block_admin/_alternance.html.twig') }}-            {{ include('administration/block_admin/_projet.html.twig') }}-            {{ include('administration/block_admin/_qualite.html.twig') }}-            {{ include('administration/block_admin/_article.html.twig') }}-            {{ include('administration/block_admin/_document.html.twig') }}+{#            {% if user_data.departement.optStage %}#}Pourquoi commenté ?—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you authored the thread.Message ID: ***@***.***> Sans virus.www.avast.com

**Comment by CyndelHerolt** [comment]
Voila c’est corrigé, j’ai PR un peu trop tôt désolée  Envoyé à partir de Courrier pour Windows De : DavidEnvoyé le :Saturday, June 3, 2023 10:04 PMÀ : Dannebicque/intranetV3Cc : CyndelHerolt; AuthorObjet :Re: [Dannebicque/intranetV3] Refonte partie Administration + Complétion Controller API UniFolio (PR ***@***.*** commented on this pull request.In templates/administration/block_admin/_etudiant.html.twig:> -                        {{ 'adm.statistiques.menu'|trans }}</a>++                <div class="col-lg-3 col-10 d-grid d-block mb-2 etudiant_bloc">+                    <div class="admin_links_bloc_header">+                        <i class="fa-solid fa-user-graduate fa-2x"></i>+                        <h4>Formation Initiale</h4>+                        <p>Gestion des absences, des notes, des étudiants, des fins de semestres,+                            des+                            stages</p>+                    </div>+                    <hr>+                    {% for semestre in user_data.semestres|filter(semestre => semestre.actif == true) %}++                        {% if semestre.annee.optAlternance == false %}++                            <a href="{{ path('administration_etudiant_parcours_semestre_index', {'semestre': semestre.id } )}}">{{ semestre.displayAvecTypeDiplome }}</a>                            href="{{ path('administration_semestre_index', {semestre:semestre.id}) }}"In templates/administration/index.html.twig:> @@ -31,16 +31,16 @@             {{ include('administration/block_admin/_ppn.html.twig') }}             {{ include('administration/block_admin/_edt.html.twig') }} -            {% if user_data.departement.optStage %}-                {{ include('administration/block_admin/_stage.html.twig') }}-            {% endif %}-            {{ include('administration/block_admin/_alternance.html.twig') }}-            {{ include('administration/block_admin/_projet.html.twig') }}-            {{ include('administration/block_admin/_qualite.html.twig') }}-            {{ include('administration/block_admin/_article.html.twig') }}-            {{ include('administration/block_admin/_document.html.twig') }}+{#            {% if user_data.departement.optStage %}#}Pourquoi commenté ?—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you authored the thread.Message ID: ***@***.***> Sans virus.www.avast.com

**Pull request merged**